blocks/init/src/Blocks/custom/tabs/components/tabs-panel.js

```diff
@@ -0,0 +1,39 @@
+import { selector } from '@eightshift/frontend-libs/scripts/helpers';
+import React from 'react';
+import manifest from './../manifest.json';
+
+export const TabsPanel = (props) => {
+
+    const { childBlocks, activeTab, setAttributes } = props;
+    const tabItems = childBlocks.map((block) => 
+        {
+            let classes = selector(manifest.tabPanelItemElement, manifest.blockClass, manifest.tabPanelItemElement);
+            if(activeTab === block.tabId) classes += ` ${manifest.tabPanelItemActiveClass}`;
```

Comment:
No inline conditionals 😬 

Also, I think you can do this using `classnames`

```js
const activeTabClass = classnames([
	activeTab === block.tabId ? manifest.tabPanelItemActiveClass : ''
]);
```

Then put this in the `className` below 🙂